website/content/docs/discovery/checks.mdx

```diff
@@ -273,6 +273,43 @@ check = {
 
 </CodeTabs>
 
+### OSService check ((#osservice-interval))
```

Comment:
The `((#osservice-interval))` part should be removed. That's only used to create backwards compatibility with an old anchor tag (if a section is renamed). This section never previously existed, so there's no need for an alias!

```suggestion
### OSService check
```